Expensify/App
# fix: login page error

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/5821

### Tests | QA Steps
1. Open the Login page.
2. Try the wrong phone number or without country code. You should get a relevant error.
3. Try any text or wrong email. You should see a relevant error.

### Tested On


### Screenshots


#### Web | Mobile Web | Desktop | Android | IOS


https://user-images.githubusercontent.com/24370807/137406029-9e8091aa-8853-4ba1-8205-ee8a7acd9a94.mp4
